### PR TITLE
Package asli.0.1

### PR DESCRIPTION
**Pull request by alastairreid**
### `asli.0.1`
Interpreter for Arm's Architecture Specification Language (ASL)
Interpreter for the language ASL that Arm uses to write their ISA specifications.

Currently supports reading/typechecking the specification,
interactive execution of ASL statements and expressions,
executing opcodes one at a time,
loading ELF files and executing Arm binaries.

---
* Homepage: https://github.com/alastairreid/asl-interpreter
* Source repo: git+https://github.com/alastairreid/asl-interpreter.git
* Bug tracker: https://github.com/alastairreid/asl-interpreter/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/asli/asli.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+maintainer: "Alastair Reid <alastair.d.reid@gmail.com>"
+authors: "Alastair Reid"
+license: "BSD3"
+homepage: "https://github.com/alastairreid/asl-interpreter"
+bug-reports: "https://github.com/alastairreid/asl-interpreter/issues"
+depends: [
+  "ocaml" {>= "4.09"}
+  "linenoise"
+  "menhir"
+  "ocamlfind"
```
Comment (kit-ty-kate) [inline packages/asli/asli.0.1/opam]:
```suggestion
  "ocamlfind" {build}
```

Comment (kit-ty-kate) [reply packages/asli/asli.0.1/opam]: It seems like you are using ocamlfind only during compilation and not as a library. This should be tagged as a build dependency

File: packages/asli/asli.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+loading ELF files and executing Arm binaries."""
+maintainer: "Alastair Reid <alastair.d.reid@gmail.com>"
+authors: "Alastair Reid"
+license: "BSD3"
+homepage: "https://github.com/alastairreid/asl-interpreter"
+bug-reports: "https://github.com/alastairreid/asl-interpreter/issues"
+depends: [
+  "ocaml" {>= "4.09"}
+  "linenoise"
+  "menhir"
```
Comment (kit-ty-kate) [inline packages/asli/asli.0.1/opam]:
```suggestion
  "menhir"
  "ocamlbuild" {build}
```
This package seems to use `ocamlbuild` and is not listed in the dependencies

File: packages/asli/asli.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "ott"
+  "pprint"
+  "zarith"
+  "z3" {<= "4.7.1"}
+]
+build: [make "INSTALL_DIR=%{prefix}%" "SHARE_DIR=%{asli:share}%" "asli"]
+install: [make "INSTALL_DIR=%{prefix}%" "SHARE_DIR=%{asli:share}%" "install"]
+remove: [
+  make "INSTALL_DIR=%{prefix}%" "SHARE_DIR=%{asli:share}%" "uninstall"
+]
```
Comment (kit-ty-kate) [inline packages/asli/asli.0.1/opam]: `remove` field is unnecessary since opam 2.0. This should be removed

Comment (alastairreid) [reply packages/asli/asli.0.1/opam]: Done.
Thanks for the help with this - I appreciate it.

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**